docs/source/admin_guide/argo-workflows.md

```diff
@@ -1,26 +1,18 @@
 # Argo Workflows
```

Comment:
These docs are deprecated. The new documentation repo is here. 

The Argo docs haven't been filled out yet - [this issue](https://github.com/nebari-dev/nebari-docs/issues/49) and [this issue ](https://github.com/nebari-dev/nebari-docs/issues/48). I can move the changes you made over to the new documentation if you'd like.

Reply:
That would be great, thanks